### PR TITLE
Line of sight readme improvements

### PR DESCRIPTION
**Pull request by tTakeda09**
**Sample**

Line of sites

**Summary**

- Add how to use the sample section
- Update how it works section and include the links to docs
- Update the sample title according to the DevEx team's convention. 
- align data section with Unity one

**Tests**

read

**ArcGIS Maps SDK Version**

### CODE REVIEW

**Review by mrnelson6 — COMMENTED** [review]
Why does the image size have to change?

**Review by mrnelson6 — APPROVED** [review]

File: sample_project/Content/SampleViewer/Samples/LineOfSight/readme.md [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
 
 Determine if the line of sight is obstructed by any object in the scene.
 
 ![Image of line of sight](LineOfSight.jpg)
 
+## How to use the sample
+
+1. Open **LineOfSight** level.
+2. Click on **ArcGISMapActor** in the Outliner.
+3. Set your API key in the Details panel.
```
Comment (avr757) [inline sample_project/Content/SampleViewer/Samples/LineOfSight/readme.md:11]:
```suggestion
3. Set your API key in the **Details** panel.
```

File: sample_project/Content/SampleViewer/Samples/LineOfSight/readme.md [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
+# Calculate line of sight
 
 Determine if the line of sight is obstructed by any object in the scene.
 
 ![Image of line of sight](LineOfSight.jpg)
 
+## How to use the sample
+
+1. Open **LineOfSight** level.
+2. Click on **ArcGISMapActor** in the Outliner.
```
Comment (avr757) [inline sample_project/Content/SampleViewer/Samples/LineOfSight/readme.md:10]:
```suggestion
2. Click on **ArcGISMapActor** in the **Outliner**.
```
Usually these are bold in our doc. Do we need to align those here also?

**Review by avr757 — APPROVED** [review]

**Review by Jade-JadeH — APPROVED** [review]
one typo and good to go

**Pull request merged**